example/spec/ingestGranule/IngestUMMGSuccessSpec.js

```diff
@@ -299,6 +299,19 @@ describe('The S3 Ingest Granules workflow configured to ingest UMM-G', () => {
       expect(onlineResources.length).toEqual(validResources.length);
     });
 
+    it('updates the CMR metadata online resources with s3credentials location', () => {
+      const s3CredentialsURL = path.join(process.env.DISTRIBUTION_ENDPOINT, 's3credentials');
+      console.log('s3CredentialsURL: ', s3CredentialsURL);
+      expect(resourceURLs.includes(s3CredentialsURL)).toBe(true);
+    });
+
+    it('adds s3credentials as the correct type to online access urls', () => {
+      console.log(JSON.stringify(onlineResources));
+      const s3CredentialsResource = onlineResources.filter((resource) => resource.Type === 'USE SERVICE API');
```

Comment:
There's nothing implicitly guaranteeing that the resource with a `Type` of `USE SERVICE API` is the s3 credentials URL, is there? 

If we wanted to guarantee we're validating the s3 credentials resource, we could add a check for the URL in addition to Type in the `.filter` callback. In which case, this test and the above test could be combined. This definitely isn't essential though.

Reply:
Good point. Checking against the url instead of just the type is done in the postToCmr portion of `parallel/ingestGranule/IngestGranuleSuccessSpec.js`. I think that method is more maintainable/extendable.